{{cookiecutter.repo_name}}/Makefile

```diff
@@ -17,6 +17,7 @@ help:
 	@echo "  pull_translations          pull translations from Transifex"
 	@echo "  push_translations          push source translation files (.po) from Transifex"
 	@echo "  quality                    run PEP8 and Pylint"
+	@echo "  pii_check                  check for PII annotations on all Django models"
```

Comment:
So, no `pii_clean` target?  I was thinking the PII report cleanup could just be lumped into the `clean` target, what do you think?

Reply:
I'll add a clean-up target.